make/Bundles.gmk

```diff
@@ -242,7 +242,10 @@ ifneq ($(filter product-bundles% legacy-bundles, $(MAKECMDGOALS)), )
       )
 
   JDK_SYMBOLS_BUNDLE_FILES := \
-      $(call FindFiles, $(SYMBOLS_IMAGE_DIR))
+      $(filter-out \
+          %.stripped.pdb, \
+          $(call FindFiles, $(SYMBOLS_IMAGE_DIR)) \
+      )
```

Comment:
Why filter *.stripped.pdb? Because they are useless for debugging? Then it should be mentioned as a comment.

Reply:
This is fixing a small issue that is independent of JEP 493. Currently the JDK/JRE bundles contain the lib.stripped.pdb files renamed to lib.pdb. The symbols bundle contains these as well but it should have the full lib.pdb files. So, by filtering out *stripped.pdb during the creation of the symbols bundle, we make sure that the full pdb files end up in the symbols bundle.